### ISSUE TITLE
make booleans true or false ... not nil

### ISSUE THREAD

**Issue by grosser** [rpush/rpush#382]
mutable_content, content_available and alert_is_json can be nil/true/false which seems dangerous ...
would you accept a PR that changes them via `change_column_null false` ?

FYI found via:

```Ruby
  it "does not have 3-state booleans (nil/false/true)" do
    bad = File.read("db/schema.rb").scan(/\st\.boolean.*/).reject { |l| l .include?(" null: false") }
    assert bad.empty?, "Boolean columns missing a default or null: false\n#{bad.join("\n")}"
  end
```

**Comment by aried3r** [comment]
Sure, I don't see why not. Would you change the migration templates or add a `3_0_1_updates` migration? Asking because I'm not sure if this should be seen as a backwards incompatible breaking change which would demand a 4.0 according to semver. I don't know if people rely on it being `nil` at any point.

In theory it should be an implementation detail which should not break stuff.

PS: That's a neat test, got some more of those sanity checks online?

**Comment by grosser** [comment]
Was thinking about adding a new migration to make it backwards compatible ... doubt it will break anything.

other tests are here https://github.com/zendesk/samson/blob/master/test/integration/cleanliness_test.rb ... few reusable ... most very project specific :)

**Comment by aried3r** [comment]
Fixed in #384
